app/src/main/res/layout-land/question_player_fragment.xml

```diff
@@ -84,23 +84,50 @@
       <ProgressBar
         android:id="@+id/question_progress_bar"
         style="@style/Widget.AppCompat.ProgressBar.Horizontal"
-        android:layout_width="match_parent"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_marginStart="30dp"
-        android:layout_marginEnd="30dp"
+        android:layout_marginEnd="54dp"
         android:layout_marginBottom="5dp"
         android:max="100"
         android:progress="@{viewModel.progressPercentage}"
         android:progressDrawable="@drawable/progress_bar"
         app:layout_constraintBottom_toTopOf="@id/question_progress_text"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent" />
+        app:layout_constraintStart_toEndOf="@id/hints_and_solution_fragment_container" />
```

Comment:
For landscape this should not be constrained.
Updated this to `app:layout_constraintStart_toStartOf="parent" />`
and marginStart to `54dp`.

This way the progress bar will always look in center.

Also, because the size of the `hints_and_solution_fragment_container ` is not more than `54dp` it will never overlap in landscape mode.

Reply:
done

Reply:
This is still not correct. As per above comment, change the constraint to
`app:layout_constraintStart_toStartOf="parent"`
and remove 
`app:layout_constraintStart_toEndOf="@id/hints_and_solution_fragment_container"`